src/main/resources/connector/org.codeoverflow.chatoverflow.requirement.service.file.fileconnector.xml

```diff
@@ -0,0 +1,8 @@
+<connector>
+    <display>File</display>
+    <description>The file service allows you to perform file operations in the data folder. These operations include reading and writing to the data folder and creating sub-folders.</description>
+    <wiki>https://github.com/codeoverflow-org/chatoverflow/wiki/File</wiki>
```

Comment:
That link doesn't work 😕. Could you please add the wiki page to make it working? 😉 😂

Reply:
I already created an issue regarding the missing pages (see #121 )